code/modules/food/recipes_microwave.dm

```diff
@@ -1476,6 +1476,14 @@
 		)
 	result = /obj/item/weapon/reagent_containers/food/snacks/swimmingcarp_spicy
 
+/datum/recipe/blethernoodlesoup
+	reagents = list(SACID = 10)
```

Comment:
Consider using SACIDS instead, so you can also make this with Formic Acid (from regular nettles).

Reply:
Done